config/detekt.yml

```diff
@@ -319,7 +319,7 @@ style:
     ignoreEnums: false
   MaxLineLength:
     active: true
-    maxLineLength: 120
+    maxLineLength: 150
```

Comment:
There were cases in the codebase where lines of code were just below `150` and made sense to be on one line than two. 
150 seemed a reasonable length of characters, I can change this back if we need to (will have to change multiple lines of code though)